src/components/tl/ucp/allgather/allgather_knomial.c

```diff
@@ -54,22 +55,21 @@
 
 void ucc_tl_ucp_allgather_knomial_progress(ucc_coll_task_t *coll_task)
 {
-    ucc_tl_ucp_task_t     *task      = ucc_derived_of(coll_task,
-                                                      ucc_tl_ucp_task_t);
-    ucc_coll_args_t       *args      = &TASK_ARGS(task);
-    ucc_tl_ucp_team_t     *team      = TASK_TEAM(task);
-    ucc_kn_radix_t         radix     = task->allgather_kn.p.radix;
+    ucc_tl_ucp_task_t *    task  = ucc_derived_of(coll_task, ucc_tl_ucp_task_t);
```

Comment:
suggestion is not to reformat existing code (* symbol), because clang format formats different depending on version, so it is better to leave as before. @samnordmann @Sergei-Lebedev what do you think?

Reply:
indeed, clang format often does this undesirable reformatting... We should have a long-term fix -- in the meantime I suggest to manually place the "*" where they should